azure/functions/durable_functions.py

```diff
@@ -39,6 +39,36 @@ def has_implicit_output(cls) -> bool:
         return True
 
 
+class EnitityTriggerConverter(meta.InConverter,
+                              meta.OutConverter,
+                              binding='entityTrigger',
+                              trigger=True):
+    @classmethod
+    def check_input_type_annotation(cls, pytype):
+        return issubclass(pytype, _durable_functions.EntityContext)
+
+    @classmethod
+    def check_output_type_annotation(cls, pytype):
+        # Implicit output should accept any return type
+        return True
+
+    @classmethod
+    def decode(cls,
+               data: meta.Datum, *,
+               trigger_metadata) -> _durable_functions.EntityContext:
+        return _durable_functions.EntityContext(data.value)
```

Comment:
Could it be possible that the **data** from the EntityContext trigger is set to None? We skip the None check in OrchestrationContext since we know it never returns None. Just want you to double check 👍 Also, I would like to know more on the EntityContext use case.

Reply:
As per our discussion, I'll keep this as-is. I don't believe `data` can be set to None, as it should be very much like the OrchestrationContext

Reply:
For this, let's have an offline discussion.

> Also, I would like to know more on the EntityContext use case.